src/icons/__generated/CoinbaseColor.tsx

```diff
@@ -6,26 +6,26 @@
 /* exported and autogenerated using SVGR from https://www.figma.com/file/ZHiNW0Wf2bj2VHVIqpjMuh/Design-Library?node-id=495%3A629 */
 import { SvgIcon, SvgIconProps } from '../../ui/SvgIcon/SvgIcon';
 const css = {
-  '& path': {
-    stroke: 'none',
-  },
+  // '& path': { stroke: 'none' }
+  // '& path': { stroke: 'none' }
 };
-export const SvgHamburger = (props: SvgIconProps) => (
+export const SvgCoinbaseColor = (props: SvgIconProps) => (
```

Comment:
These rename diffs are bizarre, the file is renamed to something totally diff??

Reply:
that is really bizarre.  why would this happen?